dags/walg_backup.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+"""
+# DAG to make a full backup of a postgres database to a gcs bucket.
+"""
+from airflow import DAG
+from datetime import datetime, timedelta
+from airflow.operators.python_operator import PythonOperator
+from dags.exec_in_pod import exec_in_pod
+
+import os
+import json
+
+YESTERDAY = datetime.now() - timedelta(days=1)
+
+namespace = os.getenv('NAMESPACE')
+in_cluster = os.getenv('LOCAL_AIRFLOW', False) == 'False'
+
+default_args = {
+    'owner': 'airflow',
+    'depends_on_past': False,
+    'start_date': YESTERDAY,
+    'email_on_failure': False,
```

Comment:
Let's add a card in the backlog to setup email for failed dags

Reply:
https://youtrack.button.is/issue/GGIRCS-1632